src/v/storage/tests/utils/disk_log_builder.cc

```diff
@@ -158,11 +158,16 @@ disk_log_builder::apply_retention(gc_config cfg) {
     return get_disk_log_impl().do_gc(cfg);
 }
 
-ss::future<> disk_log_builder::apply_compaction(
+ss::future<> disk_log_builder::apply_adjacent_merge_compaction(
   compaction_config cfg, std::optional<model::offset> new_start_offset) {
     return get_disk_log_impl().adjacent_merge_compact(cfg, new_start_offset);
 }
 
+ss::future<> disk_log_builder::apply_sliding_window_compaction(
+  compaction_config cfg, std::optional<model::offset> new_start_offset) {
+    return get_disk_log_impl().sliding_window_compact(cfg, new_start_offset);
```

Comment:
`sliding_window_compact` returns `future<bool>` so it needs something like `ignore_result` at the end

Reply:
Oops. That's what I get for walking away before my build finishes 

Pushed